scripts/process/process.counties.R

```diff
@@ -26,9 +34,11 @@ process.storm_counties <- function(viz = as.viz('storm-counties')){
                          onmouseout = "hovertext(' ');", 
                          stringsAsFactors = FALSE) %>% 
     left_join(precip.classes) %>% 
-    mutate(id = county_map_name_2_id(polyname)) %>% mutate(class = paste0(base.class, class)) %>% select(-polyname, -base.class)
+    mutate(id = county_map_name_2_id(polyname)) %>% mutate(class = paste0(base.class, class)) %>% 
```

Comment:
The data here didn't end up getting trimmed in the same way as the polygons, so this results in the county color animation being scrambled.